x-pack/plugins/canvas/common/lib/constants.ts

```diff
@@ -45,3 +45,4 @@ export const CANVAS_EMBEDDABLE_CLASSNAME = `canvasEmbeddable`;
 export const CONTEXT_MENU_TOP_BORDER_CLASSNAME = 'canvasContextMenu--topBorder';
 export const API_ROUTE_FUNCTIONS = `${API_ROUTE}/fns`;
 export const ESSQL_SEARCH_STRATEGY = 'essql';
+export const HEADER_BANNER_HEIGHT = 32;
```

Comment:
**Suggestion:** This is number can easily get out sync between the JS and Sass versions. I'd highly suggest moving it a constant that lives within the actual banner service and imported by Canvas. Then in **both** places (JS & CSS), add a comment that mentions where it's counterpart lives. Something like:

```suggestion
export const HEADER_BANNER_HEIGHT = 32; // This value also declared in Sass file `.../path/to/_bases.css`
```

Reply:
@pgayvallet: Thoughts on Caroline's suggestion above to move the banner height to live in the banner service? Sounds good to me, but would likely need your guidance on how best to implement.

Reply:
It definitely makes sense, but I think this can be done by my team as a follow-up. I created https://github.com/elastic/kibana/issues/111688

Reply:
Sounds good. In the mean time, I'll add those suggested comments in the current variable locations.